### PR TITLE
<DatePicker />, <Calendar /> Allow setting first day of the week for calendar

### PR DESCRIPTION
**Pull request by alexandraseri**
### 🔦 Summary

`<DatePicker />`, `<Calendar />` firstDayOfWeek prop

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [X] Story
  - [X] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [X] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by alexandraseri** [comment]
fixes #5477

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5765/

**Comment by alexandraseri** [comment]
> Hi @alexandraseri ,
> Thanks for contribution.
> Please add the new prop to the `d.ts` and `tsx` files of both components.

@TalZa1810  Done

File: src/Calendar/index.d.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export interface CalendarProps {
   showMonthDropdown?: boolean;
   shouldCloseOnSelect?: boolean;
   locale?: string | { distanceInWords?: {}; format?: {} };
+  firstDayOfWeek?: number
```
Comment (EgoziE) [inline src/Calendar/index.d.ts:24]: Please be more specific.
```suggestion
  firstDayOfWeek?: 1 | 2 | 3 | 4 | 5 | 6;
```
I think this will work

Comment (EgoziE) [reply src/Calendar/index.d.ts:24]: maybe it's 0 to 5

Comment (alexandraseri) [reply src/Calendar/index.d.ts]: Fixed :) it is indeed 0-6

**Pull request merged**